frontend/Dockerfile

```diff
@@ -14,5 +14,6 @@ RUN pip install --upgrade pip && \
 COPY streamlit_app.py .
 COPY ./utils ./utils
 COPY ./assets ./assets
+COPY ../common ./common
```

Comment:
You might have to modify the docker-compose context for frontend.

```
frontend:
  build:
    context: ..
    dockerfile: ./frontend/Dockerfile
  container_name: "orassistant-frontend"
  ports:
    - "8501:8501"
  networks:
    - orassistant-network
```

Reply:
oh , i believe that is what is causing the pipeline to break ,hmm, thanks for pointing this out